ROLES.md

```diff
@@ -242,7 +274,89 @@ The following apply to the area / component for which one would be an owner.
 
   - Fixing bugs.
 
-  - Mentoring and guiding approvers, members, and contributors.
+  - Identifying needed enhancements / areas for improvement / etc.
+
+  - Execute pay-down of technical debt.
+
+- Mentoring and guiding approvers, members, and new contributors.
+
+## Working Group Execution Lead
+
+Working group execution leads, or just ‘execution leads’, are responsible for
+the overall health and execution of the working group itself. Execution leads
+work with tech leads to ensure that the working group is making progress toward
+its goals, is aligned with the project roadmap, etc. The execution lead may also
+be the tech lead in a smaller working group, but they are distinct roles.
+
+### Requirements
+
+- Participant in the working group for at least 3 months, for example as scribe
+  or approver.
+
+- Recognized as having expertise in the group’s subject matter.
+
+- Sponsored by the technical oversight committee.
+
+### Responsibilities and privileges
```

Comment:
Feels like this role should figure prominently in our overall release story.  Helping to put together our release notes (and maybe even blog, if we can get that going again?).